### ISSUE TITLE
Question: Method problems

### ISSUE THREAD

**Issue by zluo01** [ta4j/ta4j#281]
I am new to ta4j library, and I am confused with several method usages. When initial the manager class object with only series, it will initial order type as BUY, does it mean it only simulate buying situation and no selling simulation? I also do not understand the amount parameter in operate method in TradingRecord. In the example, in the run method, it use Decimal.NaN. So how does it compute the overall profit then??

Thank you

**Comment by team172011** [comment]
> does it mean it only simulate buying situation and no selling simulation?

Yes, if you run the TimeSeriesManager with `OrderType.Buy` (default) your strategies will be testet with only **long** trades. If you run it with` OrderType.Sell` it will be tested with only **short** trades. The current design of the whole backtesting logic only allows to be "in the market" or not.

> I also do not understand the amount parameter in operate method in TradingRecord. In the example, in the run method, it use Decimal.NaN. So how does it compute the overall profit then??

(At the mometnt) The amount parameter will only be stored in `Order `class and is not taken into account from the criterions. Please note that the `TotalProfitCriterion ` calculates a relative profit value. A value of 1.1 means 10% profit and a value of 0.9 means 10% loss
